gaphor/storage/storage.py

```diff
@@ -139,7 +139,9 @@ def load_elements_generator(elements, factory, modeling_language, gaphor_version
     # The elements are iterated three times:
     size = len(elements) * 3
 
-    def update_status_queue(_n=[0]):
+    def update_status_queue(_n=None):
+        if _n is None:
+            _n = [0]
```

Comment:
This change causes the progress bar to stop increasing at around 50%. Not sure why, yet.

I agree that the list should be removed as argument.

To see it in action: open gaphor, from the greeter load `UML.gaphor` model. You'll see the progressbar stop  around the middle. On `main` it works fine.